src/main/java/com/redhat/devtools/lsp4ij/LanguageServerManager.java

```diff
@@ -0,0 +1,279 @@
+
+        private boolean forceStart;
+
+        private boolean enableAndStart;
+
+        public StartOptions() {
+            setEnableAndStart(true)
+                    .setForceStart(false);
+        }
+
+        /**
+         * Returns true if the language server should be started even if there are no open files corresponding to the language server and false otherwise.
+         *
+         * @return true if the language server should be started even if there are no open files corresponding to the language server and false otherwise.
+         */
+        public boolean isForceStart() {
+            return forceStart;
+        }
+
+        /**
+         * Set true if the language server should be started even if there are no open files corresponding to the language server and false otherwise.
+         *
+         * @param forceStart true if the language server should be started even if there are no open files corresponding to the language server and false otherwise.
+         * @return the start options instance.
+         */
+        public StartOptions setForceStart(boolean forceStart) {
+            this.forceStart = forceStart;
+            return this;
+        }
+
+        /**
+         * Returns true if the enabled state must be set before startup and false otherwise.
+         *
+         * <p>
+         * If enable and start is set to false, and the language server is disabled, the start method will throw  {@link LanguageServerException}.
+         * </p>
+         *
+         * @return true if the enabled state must be set before startup and false otherwise.
+         */
+        public boolean isEnableAndStart() {
+            return enableAndStart;
+        }
+
+        /**
+         * Set true if the enabled state must be set before startup and false otherwise.
+         *
+         * <p>
+         * If enable and start is set to false, and the language server is disabled, the start method will throw  {@link LanguageServerException}.
+         * </p>
+         *
+         * @param enableAndStart true if the enabled state must be set before startup and false otherwise.
+         * @return the start options.
+         */
+        public StartOptions setEnableAndStart(boolean enableAndStart) {
+            this.enableAndStart = enableAndStart;
+            return this;
+        }
+    }
+
+    /**
+     * The stop options.
+     */
+    public static class StopOptions {
+
+        public static final StopOptions DEFAULT = new StopOptions();
+
+        private boolean stopAndDisable;
+
+        public StopOptions() {
+            setStopAndDisable(true);
+        }
+
+        /**
+         * Returns true if the language server must be disabled after stopping the language server and false otherwise.
+         *
+         * @return true if the language server must be disabled after stopping the language server and false otherwise.
+         */
+        public boolean isStopAndDisable() {

```

Comment:
willDisable() ,it's implied it will be stopped

Reply:
I renamed to isWillDisable (like LSP4K which have getWillCreate which returns Boolean)